public/openapi-3-v3.1.json

```diff
@@ -3741,11 +3741,20 @@
                     "ibm"
                   ],
                   "type": "string"
+                },
+                "app_creation_workflow": {
+                  "description": "The way the source is going to be created: manually, or using superkey?",
+                  "enum": [
+                    "manual_configuration",
+                    "account_authorization"
+                  ],
+                  "type": "string"
                 }
               },
               "required": [
                 "name",
-                "source_type_name"
+                "source_type_name",
+                "app_creation_workflow"
```

Comment:
I don't think we need to mark this as required - it will default to `manual_configuration` in the backend.

Reply:
Makes sense, I'll remove it